tests/test_data/potatorolls.com/potatorolls_2.json

```diff
@@ -0,0 +1,46 @@
+{
+  "author": "Martin's Famous Potato Rolls and Bread",
+  "canonical_url": "https://potatorolls.com/recipes/pineapple-upside-down-sticky-buns/",
+  "host": "potatorolls.com",
+  "image": "https://potatorolls.com/wp-content/uploads/Upside-Down-Pineapple6-960x640.jpg",
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "12 Martin’s Sweet Dinner Potato Rolls",
+        "½ Cup Butter",
+        "1 Cup Brown Sugar",
+        "1 20-Ounce Can Sliced Pineapple, drained",
+        "1 8-Ounce Jar Maraschino Cherries"
+      ],
+      "purpose": null
+    }
+  ],
+  "ingredients": [
+    "12 Martin’s Sweet Dinner Potato Rolls",
+    "½ Cup Butter",
+    "1 Cup Brown Sugar",
+    "1 20-Ounce Can Sliced Pineapple, drained",
+    "1 8-Ounce Jar Maraschino Cherries"
+  ],
+  "instructions": "Preheat oven to 350 F.\nGrease a 9” baking dish or pan. (A square springform pan works well.)\nPat slices of pineapple dry between sheets of paper towels.\nArrange pineapple slices evenly in bottom of pan.\nPlace a cherry in the center of each pineapple ring.\nIn a medium saucepan over medium heat, melt butter and brown sugar, stirring until sugar is dissolved.\nCarefully pour butter mixture over the pineapple and cherries.\nPlace Sweet Dinner Potato Rolls top-down on the pineapple and cherries.\nWith a knife, poke holes at the corners of the rolls.\nCover with foil and bake for 8-10 minutes.\nRemove from oven and allow to cool for 5 minutes.\nCarefully turn the pan upside down onto a serving platter.\nServe and enjoy!",
+  "instructions_list": [
+    "Preheat oven to 350 F.",
+    "Grease a 9” baking dish or pan. (A square springform pan works well.)",
+    "Pat slices of pineapple dry between sheets of paper towels.",
+    "Arrange pineapple slices evenly in bottom of pan.",
+    "Place a cherry in the center of each pineapple ring.",
+    "In a medium saucepan over medium heat, melt butter and brown sugar, stirring until sugar is dissolved.",
+    "Carefully pour butter mixture over the pineapple and cherries.",
+    "Place Sweet Dinner Potato Rolls top-down on the pineapple and cherries.",
+    "With a knife, poke holes at the corners of the rolls.",
+    "Cover with foil and bake for 8-10 minutes.",
+    "Remove from oven and allow to cool for 5 minutes.",
+    "Carefully turn the pan upside down onto a serving platter.",
+    "Serve and enjoy!"
+  ],
+  "language": "en-US",
+  "yields": "12 servings",
+  "site_name": "Martin's Famous Potato Rolls and Bread",
+  "title": "Pineapple Upside Down Sticky Buns",
+  "total_time": 20
```

Comment:
This does seem correct; commenting because the source data appears to be in slightly-malformed ISO8601 format (`PT20 Mins`) - but we do get the expected output here.

Reply:
Must be only handling up till the space and registering it as a valid input, not necessarily a bad thing

Reply:
I thought so too, but it seems that it's due to the upper-case capital `M` / `Mins` after the space.  And that's OK I think.